src/main/resources/org/biouno/unochoice/stapler/unochoice/unochoice.js

```diff
@@ -222,139 +221,6 @@ var UnoChoice = UnoChoice || ($ => {
                     _self.getFilterElement().setOriginalArray(originalArray);
                 }
             } else if (parameterElement.tagName === 'DIV' || parameterElement.tagName === 'SPAN') {
-                if (parameterElement.children.length > 0 && parameterElement.children[0].tagName === 'TABLE') {
```

Comment:
This is an example of code we could remove since moving the min-jenkins version past `2.277.1`